CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ set(JSON_BuildTests OFF CACHE INTERNAL "")
 set(JSON_Install OFF CACHE INTERNAL "")
 set(JSON_ImplicitConversions OFF CACHE INTERNAL "")
 
+if (WIN32)
```

Comment:
```suggestion
if (static AND WIN32)
```

Reply:
Take my suggestion as a note. It's in case you want to apply the code to the `mumble` repository.

Reply:
I think I am tending more into the direction of
```suggestion
if (static)
```
and drop the Windows check entirely :thinking:

Reply:
Good idea.